nservicebus/sqlserver/operations-scripting.md

```diff
@@ -90,3 +90,44 @@ snippet:sqlserver-return-to-source-queue
 ### Using the retry helper methods
 
 snippet:sqlserver-return-to-source-queue-usage
+
+
+### How to shift SqlTransport audit log to long term storage
```

Comment:
This instruction is particular to SQL...there's another issue for archiving MSMQ audits so there would need to be a distinction in the title. The 'long term storage' is suggesting that you are not really ever going to put them back in your production database.

You are right though, the title is a little clumsy sounding.